commit 608697149c8f141837d28fec707949386f52216b (refs/changes/32/3732/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-03-27T17:01:43-07:00 (1 year, 6 months ago)
    
    TRITON-282 drop obsolete gsed'ing of IMGAPI SMF manifest at zone setup

diff --git a/boot/setup.sh b/boot/setup.sh
index 565a253..8f1fb0c 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -40,7 +40,6 @@ if [[ ! -d $LOCAL_BASE_DIR ]]; then
 fi
 chown nobody:nobody $LOCAL_BASE_DIR
 
-$(/opt/local/bin/gsed -i"" -e "s/@@PREFIX@@/\/opt\/smartdc\/imgapi/g" /opt/smartdc/imgapi/smf/manifests/imgapi.xml)
 /usr/sbin/svccfg import /opt/smartdc/imgapi/smf/manifests/imgapi.xml
 
 # Log rotation.
